packages/components/tree-v2/src/composables/useTree.ts

```diff
@@ -202,7 +204,18 @@ export function useTree(
   }
 
   function setExpandedKeys(keys: TreeKey[]) {
-    expandedKeySet.value = new Set(keys)
+    const expandedKeys = new Set<TreeKey>()
+    const nodeMap = tree.value?.treeNodeMap || new Map()
```

Comment:
```suggestion
    const nodeMap = tree.value!.treeNodeMap
```
`tree.value` never be undefined when setExpandedKeys is called.

Reply:
Thank you for the code review